src/test/components/MarkerChart.test.js

```diff
@@ -232,14 +232,24 @@ describe('MarkerChart', function () {
     );
     expect(arcOperationsXY.size).toBe(3);
 
-    // Check that we have rect operations for the first small interval marker
-    // and the later longer marker.
+    // Check that we have a fillRect operation for the first small interval
+    // marker. We filter also using w or h to filter out the initial clearing
+    // fillRect operation as well as the separators.
+    const fillRectOperations = drawCalls.filter(
+      ([operation, , , w, h]) =>
+        operation === 'fillRect' &&
+        w !== containerWidth - TIMELINE_MARGIN_RIGHT &&
+        h !== containerHeight
+    );
+    expect(fillRectOperations).toHaveLength(1);
+
+    // Check that we have a rect operation for the later longer marker.
```

Comment:
nit: "we have rect operations" or "we have 2 rect operations" could be better since there are multiple for the longer marker.

Reply:
The comment means that _this_ rect operation is the one we're interested about, even if there's a second one because of the "clip" operation like the next comment says. So I think I'll keep thsi comment, but maybe rewrite it a bit.